test/actions/linkActions.test.js

```diff
@@ -0,0 +1,39 @@
+import test from 'blue-tape'
+import * as actions from '../../src/actions/linkActions'
+
+const CALENDAR = {type: 'course', id: 'MI-RUB', date: '2015-10-11'}
+
+test('calendarUrl()', t => {
+  t.test('calendarUrl() with all parameters', st => {
```

Comment:
No, it does not. Actually, I could omit the wrapper test but I thought this is nicer.